src/config.js

```diff
@@ -13,6 +13,7 @@ const extractConfigSchema = {
     type: ['object', 'null'],
     properties: {
         output: { type: 'string' },
+        location: { type: 'string' },
```

Comment:
better to use enum option (better validation)
Just like in 'unresolved' property:
```javascript
{
        unresolved: { enum: [FAIL, WARN, SKIP] },
}
```